ui/shared/DropdownMenuItems.jsx

```diff
@@ -8,23 +8,44 @@ const DropdownMenuItems = ({
   selectedItem,
   updateData,
   options,
+  pinned = [],
   namespace = '',
 }) => (
   <DropdownMenu className="overflow-auto dropdown-menu-height">
-    {options.map(item => (
-      <DropdownItem
-        tag="a"
-        key={`${namespace}${item}`}
-        onClick={() => updateData(item)}
-      >
-        <FontAwesomeIcon
-          icon={faCheck}
-          className={`mr-1 ${selectedItem === item ? '' : 'hide'}`}
-          title={selectedItem === item ? 'Checked' : ''}
-        />
-        {item}
-      </DropdownItem>
-    ))}
+    {/* Items pinned to top of dropdown */}
+    {pinned.length > 0 &&
```

Comment:
I think we should go through the `pinned` values and only keep the ones that are also in `options`. This assures us that we don't show pinned a value that shouldn't be displayed at all.

Reply:
good point 👌